EIPS/eip-4762.md

```diff
@@ -221,15 +221,13 @@ When executing a transaction, maintain four sets:
 
 When an **access** event of `(address, sub_key, leaf_key)` occurs, perform the following checks:
 
- * Perform the following steps unless `address` is either:
-   * the transaction's originator;
-   * the transactions's target;
+ * Perform the following steps unless event is a *Transaction access event*;
```

Comment:
adress could be txsender or target and event could be some state access, so bettter to clarify the this gas exception is limited to tx events